Documentation/targets.md

```diff
@@ -1,30 +1,113 @@
 # Targets
 
+- alertmanager
+- api
+- app-exporter
+- app-operator
 - cadvisor
 - calico
 - cert-exporter
 - chart-operator
 - cluster-autoscaler
+- cluster-operator
+- cluster-service
+- companyd
+- credentiald
+- crsync
 - docker
+- draughtsman
 - etcd
+- etcd-backup-operator
+- fluentbit
+- g8s-grafana
+- g8s-prometheus
+- gatekeeper
 - kube-proxy
 - kube-state-metrics
 - kubelet
 - kubernetes-apiserver
 - kubernetes-controller-manager
 - kubernetes-scheduler
+- kubernetesd
 - net-exporter
 - nginx-ingress-controller
 - node-exporter
+- node-operator
+- organization-operator
+- prometheus-meta-operator
+- release-operator
+- tokend
+- userd
+- vault
+- vault-exporter
 
 ### AWS specific
 
 - aws-node
+- aws-operator
+- cluster-autoscaler
 
-## Known issues
+### Azure specific
 
-* docker: fails to be scrapped, due to metrics port not being exposed on giantswarm releases below 11.0.0, see [commit](https://github.com/giantswarm/k8scloudconfig/commit/6ecc07e665c3e854dfa8be102a8c6446d1d9dc3c#diff-be6122463e3fe598d118a80e09254d3d)
+- azure-operator
+- cluster-autoscaler
+
+### KVM specific
+
+- kvm-operator
+
+## Missing
+
+- bridge-operator
+- cert-operator
+- prometheus-operator-app
+- CP prometheus
+- TC prometheus
```

Comment:
What's the difference?

Reply:
That's mostly so we do not forget and because CP prometheus should be checked via heartbeats while TC prometheus should be scraped by the CP prometheus

Reply:
I don't think we're planning on scrapping TC prometheus with CP prometheus. But I'm now thinking we need a way to know when prometheuses are down.

Reply:
I thought it was what we intended 
![image](https://user-images.githubusercontent.com/2787548/91987751-be95fa80-ed2e-11ea-8b81-102ebc3ca05b.png)

Reply:
We need to scrape the TC prometheus servers from the CP prometheus server.